### PR TITLE
ADD: Information about EI instance to json

### PR DESCRIPTION
**Pull request by ned29**

### CODE REVIEW

File: src/main/java/com/ericsson/ei/controller/info/InstanceInfo.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    @Autowired
+    private ParseInstanceInfoEI istanceInfo;
+
+    @RequestMapping(value = "/information", produces = MediaType.APPLICATION_JSON_VALUE)
+    public String parseInfo() {
+        try {
+            return new ObjectMapper().writerWithDefaultPrettyPrinter().writeValueAsString(istanceInfo);
+        } catch (Exception e) {
+            LOGGER.error("Serialization is failed " + e.getMessage());
```
Comment (tobiasake) [inline src/main/java/com/ericsson/ei/controller/info/InstanceInfo.java]: Should be "Serializaion has failed".

**Comment by tobiasake** [comment]
And avoid using Ericsson User Story numbers on Commits on GitHub.

**Comment by emichaf** [comment]
Will spring.application.name and application.name be used in application.properties... ?

**Comment by ned29** [comment]
@emichaf  
I added 
build.version=@project.version@
application.name=@project.artifactId@
because Tobias said that we need this info about EI instance.

**Pull request closed without merging**